data-prepper-plugins/rds-source/src/main/resources/org/opensearch/dataprepper/transforms/templates/rds-template.yaml

```diff
@@ -48,6 +48,9 @@
         codec:
           event_json:
         default_bucket_owner: "<<FUNCTION_NAME:getAccountIdFromRole,PARAMETER:$.<<pipeline-name>>.source.rds.aws.sts_role_arn>>"
+        client:
+          max_connections: 100
```

Comment:
isn't this the default value ?

Reply:
Default is 50 for max_connections and 10s for acquire_timeout. Doubling both here.